### ISSUE TITLE
Detect if GTM is active

### ISSUE THREAD

**Issue by CiaranPark** [tmwagency/trak.js#2]
In some cases there will be need for a 'trak-this' class on an element, this will allow Richard via GTM to target all instances of the class and setup on event labels. 

##Tracking a navigation click
###Workflow 1 - GTM Active
Just add class to element (trak-nav-main)

###Workflow 2 - No GTM (custom tracking Diageo, pingdom)
Need to add data-trak

###Workflow 3 - NO gtm but potential upgrade
Requires both Class & data-attr

So basically an element will need both and our plugin will have to detect which is used.
